extensions/arc/package.json

```diff
@@ -621,27 +621,31 @@
                           "description": "%arc.sql.cores-limit.description%",
                           "variableName": "AZDATA_NB_VAR_SQL_CORES_LIMIT",
                           "type": "number",
+                          "min": 1,
                           "required": false
                         },
                         {
                           "label": "%arc.cores-request.label%",
                           "description": "%arc.sql.cores-request.description%",
                           "variableName": "AZDATA_NB_VAR_SQL_CORES_REQUEST",
                           "type": "number",
+                          "min": 1,
                           "required": false
                         },
                         {
                           "label": "%arc.memory-limit.label%",
                           "description": "%arc.sql.memory-limit.description%",
                           "variableName": "AZDATA_NB_VAR_SQL_MEMORY_LIMIT",
                           "type": "number",
+                          "min": 1,
```

Comment:
I think the memory fields should have 2Gi minimums to match the validation I see in azdata:

```
azdata arc sql mi create -n s -ml 1Gi
Memory limit must be at least 2Gi

azdata arc sql mi create -n s -mr 1Gi
Memory request must be at least 2Gi
```

Reply:
Ah fantastic catch. Thank you.